mongoengine/document.py

```diff
@@ -291,7 +288,12 @@ def save(self, force_insert=False, validate=True, clean=True,
 
         doc = self.to_mongo()
 
-        created = ('_id' not in doc or self._created or force_insert)
+        # I think the self._created flag is not necessarily required in PyMongo3
```

Comment:
Why wouldn't it be necessary?  I haven't looked at PyMongo3 but maybe a comment as to why might help.
Also does checking it cause any issues?